From 354481601e548b53936ac4a3e887293bee408683 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Tue, 9 Sep 2014 17:01:09 -0500
Subject: [PATCH] ENGR00331054 Revert "ENGR00303629-1: ARM: dts: imx6sx-sdb: 
 support NOR Flash SF25FL128S"

The DT node for QSPI has already existed, it's not necessary to cherry-pick this
commit.

This reverts commit 84fef446d673601ce1db811dbadca6df2c131fa8.

Signed-off-by: Allen Xu <b45815@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 63f8c41..ff7e473 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -546,25 +546,3 @@
 		};
 	};
 };
-
-&qspi2 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_qspi2_1>;
-	status = "okay";
-
-	flash0: s25fl128s@0 {
-		#address-cells = <1>;
-		#size-cells = <1>;
-		compatible = "spansion,s25fl128s";
-		spi-max-frequency = <66000000>;
-		reg = <0>;
-	};
-
-	flash1: s25fl128s@1 {
-		#address-cells = <1>;
-		#size-cells = <1>;
-		compatible = "spansion,s25fl128s";
-		spi-max-frequency = <66000000>;
-		reg = <1>;
-	};
-};
-- 
1.8.0

